app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -5,51 +5,57 @@ class AamvaStateMaintenanceWindow
     # All AAMVA maintenance windows are expressed in 'ET' (LG-14028),
     TZ = 'America/New_York'
 
+    Window = Data.define(:cron, :duration) do
+      def cron_val
+        Time.use_zone(TZ) { Fugit.parse_cron(cron) }
+      end
+    end.freeze
```

Comment:
I'm unsure, but does `.freeze` do anything for us here? 🤔

Reply:
I'm not sure if there is any actual benefit in terms of memory, but it has the practical effect of keeping rubocop from yelling at me with a `Style/MutableConstant` complaint. 😇